docs/openxla.md

```diff
@@ -0,0 +1,36 @@
+# OpenXLA
+
+PyTorch/XLA is a Python package that uses the [XLA deep learning
+compiler](https://www.tensorflow.org/xla) to connect the [PyTorch deep learning
+framework](https://pytorch.org/) and [Cloud
+TPUs](https://cloud.google.com/tpu/). PyTorch/XLA now pull XLA from OpenXLA. 
+OpenXLA here means [open source machine learning compiler XLA for GPUs, CPUs, and ML accelerators](https://github.com/openxla/xla), 
+which belongs to the same-name community [OpenXLA Community](https://github.com/openxla) and this community also contain [StableHLO](https://github.com/openxla/stablehlo).
```

Comment:
Let's rephrase this as:

```
As of ***<insert_date_where_we_pushed_openxla_migration>***, PyTorch/XLA now ***pulls*** XLA from OpenXLA. 
OpenXLA ***is an*** [open source machine learning compiler XLA for GPUs, CPUs, and ML accelerators](https://github.com/openxla/xla). 
```

Let's remove the sentence `which belongs to the same-name community [OpenXLA Community](https://github.com/openxla) and this community also contain [StableHLO](https://github.com/openxla/stablehlo).`  since this seems redundant.

Reply:
thanks, updated